### PR TITLE
Fixing the crash within the RequestAPI call

### PR DESCRIPTION
**Pull request by nkeerthy-okta**
### Problem Analysis (Technical)
Fixing the crash within RequestAPI call when the data is nil

### Solution (Technical)
adding guard to make sure that we would do an extra check for the nil data

### Steps to reproduce:
Passing a nil data with the get call

Actual result: App would crash

Expected result: App to handle nil data

### Tests
Adding test to check we would return nil with getJSON call when we have nil data from get call

### CODE REVIEW

**Review by stevelind-okta — APPROVED** [review]
Looks good! 

Nice to have: In your test case perhaps we should add a positive test case? You should be able to do this with a local file url.

File: Tests/RequestAPITests.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import XCTest
+
+@testable import OktaJWTLib
+class RequestAPITests: XCTestCase {
+
+    override func setUpWithError() throws {
```
Comment (anastasiiaiurok-okta) [inline Tests/RequestAPITests.swift:18]: nit: let's not keep empty functions in code

**Review by anastasiiaiurok-okta — APPROVED** [review]

File: Sources/RequestsAPI.swift [hunk 2]
```diff
@@ -77,9 +77,10 @@ open class RequestsAPI: NSObject {
      A JSON representation of the HTTP response body.
      */
     open class func getJSON(_ url: URL) -> [String: Any]? {
-        let data = self.get(url)
-
-        if let json = try? JSONSerialization.jsonObject(with: data!, options: []) as? [String: Any] {
```
Comment (IldarAbdullin-okta) [inline Sources/RequestsAPI.swift:82]: Are there any other unsafe code like that?

Comment (nkeerthy-okta) [reply Sources/RequestsAPI.swift:82]: Found one, Fixing it for another potential bug https://github.com/okta/okta-ios-jwt/blob/master/Sources/OktaJWT.swift#L255

**Review by stevelind-okta — APPROVED** [review]

**Review by IldarAbdullin-okta — APPROVED** [review]
LGTM!
